app/representers/api/distribution_representer.rb

```diff
@@ -0,0 +1,20 @@
+# encoding: utf-8
+
+class Api::DistributionRepresenter < Api::BaseRepresenter
+  property :id, writeable: false
+  property :type
+  property :guid
+  property :url
+  hash :properties
```

Comment:
If it isn't an addressable, stand alone resource, then I think this is an ok way to handle it.
If it was able to be updated distinctly from the distribution, as its own resource, then no.
Do you have any other ideas?